iceoryx_utils/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -81,18 +81,16 @@ UnixDomainSocket::UnixDomainSocket(const NoPathPrefix_t,
     else
     {
         m_maxMessageSize = maxMsgSize;
-        auto createResult = createSocket(mode);
+        auto initalizeResult = initalizeSocket(mode);
```

Comment:
The code is correct but this would be a typical use case for the `and_then`/`or_else` pattern to reduce the amount of code and future errors.